Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -272,6 +276,23 @@ void ABenchmarkGymGameModeBase::Tick(float DeltaSeconds)
 		StatStopFileTimer.SetTimer(999999);
 	}
 #endif
+	if (MemReportIntervalTimer.HasTimerGoneOff())
+	{
+		if (MemReportInterval>0) //Avoid execute memreport each tick before the worker flag take effect
+		{
+			FString Cmd = TEXT("memreport -full");
+			if (GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
```

Comment:
Do we need support native unreal mode?

Reply:
When SpatialOS is disabled, the command will follow the native Unreal naming rule as `Map-Platform-SnapshotTimestamp.memreport`. Both native and Spatial reports will be saved to the same file folder named with `Map-Platform-ProfileSessionTimestamp\`